### PR TITLE
infoschema: change table_type for system tables

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs <crazycs520@gmail.com>

### What problem does this PR solve? 

Fix Issue https://github.com/pingcap/tidb/issues/15123.

Change all table's table_type in `('INFORMATION_SCHEMA','PERFORMANCE_SCHEMA','METRICS_SCHEMA','INSPECTION_SCHEMA')` databases to `SYSTEM VIEW`.

### Check List 

Tests 

 - Unit test

Side effects

Related changes

 - Need to cherry-pick to the release branch

Release note

 - Change all system tables's `TABLE_TYPE` to `SYSTEM VIEW`.

### CODE REVIEW

**Comment by GregoryIan** [comment]
LGTM

File: infoschema/tables.go [hunk 1]
```diff
@@ -1356,13 +1356,21 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.D
 				if rowCount != 0 {
 					avgRowLength = dataLength / rowCount
 				}
+				var tableType string
+				switch schema.Name.L {
+				case util.InformationSchemaName.L, util.PerformanceSchemaName.L,
```
Comment (siddontang) [inline infoschema/tables.go]: how about MySQL? also, treats these as VIEW?

Comment (bb7133) [reply infoschema/tables.go]: For MySQL, tables in `information_schema` are SYSTEM_VIEWs, but tables in `performance_schema` is marked as `BASE`.

Theere was a discussion on this before...and we treat this as a bug of MySQL.

**Pull request merged**